lib/ably/realtime/connection.rb

```diff
@@ -55,22 +55,22 @@ class Connection
       # @return [Ably::Realtime::Connection::STATE]
       #
       STATE = ruby_enum('STATE',
-        :initialized,
-        :connecting,
-        :connected,
-        :disconnected,
-        :suspended,
-        :closing,
-        :closed,
-        :failed
+                        :initialized,
```

Comment:
formatting issue?

Reply:
Fixed !
Actually, my IDE RubyMine auto-formatted previously. 
Not sure what is a correct formatting for ruby. We don't have a ruby auto-formatting tool either : (

Reply:
It will be great if we can integrate something like https://github.com/rubocop/rubocop

Reply:
Seems we already have a pending issue for the same => https://github.com/ably/ably-ruby/issues/139

Reply:
Resolving conversation for now as per discussion in the standup 👍